clowdapp-minimal.yaml

```diff
@@ -163,12 +161,12 @@ parameters:
     value: "5"
   - name: SIDEKIQ_CONCURRENCY
     value: "1"
-  - name: RAILS_MAX_THREADS
-    value: "1"
   - name: DISABLE_RBAC
     value: "false"
   - name: PUMA_WORKERS
     value: "3"
+  - name: PUMA_MIN_THREADS
+    value: "5"
```

Comment:
Can we increase number of processes instead of increasing the minimum threads?

Reply:
The point of this change is to have minimum threads equal to maximum threads so we have some assumption on how much memory we need. After fixing the memory leak, the silent restart is happening because of ad-hoc forking of threads under high load. This setting basically prevents puma from having high fluctuations in memory usage.

I am hoping that this will increase the number of silent restarts that would help us get to a lower number of max threads (i.e. 4) or to higher memory allocation (~1.2GB).